_posts/2017-02-21-twis-93.md

```diff
@@ -27,6 +27,10 @@ This week's status updates are [here](https://www.standu.ps/project/servo/).
 - [Huxley](https://github.com/UnICorN21)
 - [Ian](https://github.com/Greeene)
 - [Sam Liu](https://github.com/samliu)
+- [Jefry Lagrange](https://github.com/zimio)
+- [Neck Varentsov](https://github.com/Varentsov)
+- [vwvww](https://github.com/vwvww)
+- [Ethan Glasser-Camp](https://github.com/glasserc]
```

Comment:
Heads up: the closing paren for the URL is a `]` instead of a `)` so it breaks the link.